src/connector/src/sink/encoder/json.rs

```diff
@@ -855,6 +855,8 @@ mod tests {
             ["schema"]
             .to_string();
         let ans = r#"{"fields":[{"field":"v1","optional":true,"type":"boolean"},{"field":"v2","optional":true,"type":"int16"},{"field":"v3","optional":true,"type":"int32"},{"field":"v4","optional":true,"type":"float"},{"field":"v5","optional":true,"type":"string"},{"field":"v6","optional":true,"type":"int32"},{"field":"v7","optional":true,"type":"string"},{"field":"v8","optional":true,"type":"int64"},{"field":"v9","optional":true,"type":"string"},{"field":"v10","fields":[{"field":"a","optional":true,"type":"int64"},{"field":"b","optional":true,"type":"string"},{"field":"c","fields":[{"field":"aa","optional":true,"type":"int64"},{"field":"bb","optional":true,"type":"double"}],"optional":true,"type":"struct"}],"optional":true,"type":"struct"},{"field":"v11","items":{"items":{"fields":[{"field":"aa","optional":true,"type":"int64"},{"field":"bb","optional":true,"type":"double"}],"optional":true,"type":"struct"},"optional":true,"type":"array"},"optional":true,"type":"array"},{"field":"12","optional":true,"type":"string"},{"field":"13","optional":true,"type":"int32"},{"field":"14","optional":true,"type":"string"}],"name":"test","optional":false,"type":"struct"}"#;
+        let schema = serde_json::from_str::<Value>(&schema).unwrap();
+        let ans = serde_json::from_str::<Value>(ans).unwrap();
         assert_eq!(schema, ans);
```

Comment:
It was due an update in the dependencies, the order of the string changed, so I changed it to use json instead of just plain string

Reply:
This was because `deno_core` adds the feature `preserve_order` for `serde_json`, I [patched](https://github.com/bakjos/deno_core/commit/70d1544ed07380d3f37a92118d5ca4f1f20c6616)  the dependency to remove it, but I'm not sure if this is the right place to do so https://github.com/risingwavelabs/risingwave/blob/cd2c0d4cb00e18f727653586c6977a6e8fda267f/Cargo.toml#L309-L310

Reply:
I'm okay with this patch. cc @xxchan any better idea on this feature interaction?